core/src/main/java/hudson/model/UpdateSite.java

```diff
@@ -1297,6 +1301,17 @@ public Plugin(String sourceId, JSONObject o) {
             dependencies = getPresizedMutableMap(depCount);
             optionalDependencies = getPresizedMutableMap(optionalDepCount);
             this.healthScore = o.has("healthScore") ? o.getInt("healthScore") : null;
+            if (healthScore != null) {
```

Comment:
This duplicates the code how to calculate the clazz from Pluginwrapper. So when we want to change the levels we can easily forget one place.

Reply:
I mutualized the code in `PluginWrapper`. Hopefully it's fine.